internal/organisation/validate_dns.go

```diff
@@ -0,0 +1,63 @@
+package organisation
+
+import (
+	"net"
+	"strings"
+)
+
+// Override for mocking purposes
+var resolver DNSResolver = &DefaultResolver{}
+
+func validateDNS(o Organisation, domain string) (bool, error) {
+	oa := strings.ToLower(o.Addr.String())
+
+	recs, err := resolver.LookupTXT("_bitmaelum." + domain)
+	if err != nil {
+		return false, err
+	}
+
+	for _, txt := range recs {
+		if strings.ToLower(txt) == oa {
```

Comment:
We are only checking that we store the hash of the organization name on a TXT address... is this enough? shouldn't be signed?

Reply:
yes.. it is enough. Because the fact that you are able to add the specific (hash) value to the DNS, proofs you are the owner of the DNS.. this is basically what most systems do when you need to proof domain ownership